Makefile

```diff
@@ -342,9 +345,13 @@ endif
 # Current operator version
 VERSION ?= 0.37.0
 
+.PHONY: generate-operator-bundle
+generate-operator-bundle: LATEST_VERSION := $(shell curl -sL https://api.github.com/repos/Azure/azure-service-operator/releases/latest  | jq '.tag_name' --raw-output )
```

Comment:
Is this the latest version or the latest tag? Above in `helm-chart-manifests` this same value is called `LATEST_TAG`, but here it's `LATEST_VERSION` and `LATEST_TAG` includes the `$(PUBLIC_REPO)` prefix. We should be consistent in what we're calling it (since as far as I can tell it's the same concept in both places?)

Reply:
I'll make them consistent.